### DISCUSSION TITLE
Is there a way to provide a translation for a specific count value when using pluralization?

### DISCUSSION THREAD

**Discussion by szymonnowak-st** [i18next/i18next#2058] [Q&A] [answered]
I've got a translation that should be different depending on whether the value of `count` is 1, 2, 3, or more. In English you only have `one` and `other` cases. Is there a way to provide a translation for a specific count value?

Example:
- You selected item "A". (`count = 1`)
- You selected items "A" and "B". (`count = 2`)
- You selected "A", "B" and 1 more item. (`count = 3`)
- You selected "A", "B" and 3 more items. (`count  >= 4`)

**Comment by jamuhl** [comment, marked as answer]
https://github.com/i18next/i18next-intervalPlural-postProcessor might help

**Reply by szymonnowak-st** [reply]
Thanks so much for such a quick reply!
